File: app/src/main/java/tourtle/ticketing2/model/UserModel.java
package tourtle.ticketing2.model;

public class UserModel {
    private String idWisata,namaWisata,tanggal,jam, status,email;
    private Integer kodeVersi;

    public UserModel(String idWisata, String namaWisata, String tanggal, String jam, String status,String email,Integer kodeVersi) {
        this.idWisata = idWisata;
        this.namaWisata = namaWisata;
        this.tanggal = tanggal;
        this.jam = jam;
        this.status = status;
        this.email = email;
        this.kodeVersi = kodeVersi;
    }

    public String getIdWisata() {
        return idWisata;
    }

    public String getNamaWisata() {
        return namaWisata;
    }

    public String getTanggal() {
        return tanggal;
    }

    public String getJam() {
        return jam;
    }

    public String getStatus() {
        return status;
    }

    public String getEmail() {
        return email;
    }

    public Integer getKodeVersi() {
        return kodeVersi;
    }
}

File: app/src/main/java/tourtle/ticketing2/model/UserModel.kt
package tourtle.ticketing2.model

class UserModel(private val idWisata: String?, private val namaWisata: String?, private val tanggal: String?, private val jam: String?, private val status: String?, private val email: String?, private val kodeVersi: Int?) {
    fun getIdWisata(): String? {
        return idWisata
    }

    fun getNamaWisata(): String? {
        return namaWisata
    }

    fun getTanggal(): String? {
        return tanggal
    }

    fun getJam(): String? {
        return jam
    }

    fun getStatus(): String? {
        return status
    }

    fun getEmail(): String? {
        return email
    }

    fun getKodeVersi(): Int? {
        return kodeVersi
    }

}